packages/backend/src/apps/delay/actions/delay-until/index.ts

```diff
@@ -11,16 +13,43 @@ export default defineAction({
       key: 'delayUntil',
       type: 'string' as const,
       required: true,
-      description: 'Delay until the date. E.g. 2022-12-18',
+      description:
+        'Delay until the date. E.g. 2023-08-25, 25 Aug 2023, 25/08/2023',
+      variables: true,
+    },
+    {
+      label: 'Delay until (Time)',
```

Comment:
If we're going to support hours in the main `delayUntil` field, then I would rather we remove this hours field and reduce complexity

Reply:
Nvm I saw the linear issue. I talk to ian about it first

Reply:
yea i talked to him about this pr alr actually

Reply:
ok